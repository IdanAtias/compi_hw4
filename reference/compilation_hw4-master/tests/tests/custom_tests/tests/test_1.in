int a;
int b = 5;

matrix c(2, 3);
matrix d(2, 3) = c;
matrix e(2, 3) = [1, 2, 3; 4, 5, 6];

[1, 2, 3; 4, 5, 6];

int f = d[1, 1];
int g = e[1, 1];

int h = d(1, 1);
