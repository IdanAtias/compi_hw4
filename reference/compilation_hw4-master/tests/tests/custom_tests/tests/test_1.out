a=
0
b=
5
c=
0,0,0
0,0,0
d=
0,0,0
0,0,0
e=
1,2,3
4,5,6
1,2,3
4,5,6
f=
0
g=
5
ScalarExpected
